example/index.js

```diff
@@ -14,11 +14,11 @@ const {
 
 const thunkMiddleware = require('redux-thunk').default;
 const syncBrainstemMiddleware = require('../lib/middleware/update-storage-manager').default;
-const loggerMiddleware = require('./middleware/logger');
+const loggerMiddleware = require('./middleware/logger').default;
 
-const Post = require('./models/post');
-const Posts = require('./collections/posts');
-const Users = require('./collections/users');
+const Post = require('./models/post').default;
```

Comment:
I'm actually a bit confused as to why we'd show our examples using require and not import at this point.

Reply:
I want to keep the change set focused on the library. 

At some point, we need to make a decision on how to move towards better patterns in these examples. I think I would prefer to redo the whole thing because a simple autocompleter does not really do it justice. Maybe we can come up with better examples with some awesome Mavenlink design system components ;)